tileview/src/main/java/com/qozix/tileview/TileView.java

```diff
@@ -790,7 +792,6 @@ public void onZoomEnd( float scale, Origination origin ) {
 
   @Override
   public void onDetailLevelChanged( DetailLevel detailLevel ) {
-    requestRender();
```

Comment:
so it's technically not useless - it's cancelled (and re-requested) in `updateTileSet` _only_ if the `DetailLevel` changes.  i'm a little hesitant to remove this - seems a little out of scope for the PR.